app/controllers/summoners_controller.rb

```diff
@@ -63,6 +63,71 @@ def find_by_id
     end
   end
 
+  def get_league_all
+    summoner_existance = League.find_by(region: params['region'], queue: params['queue'], name: params['leagueName'], tier: params['tier'], division: params['division'], summonerId: params['id'])
+    if summoner_existance.nil?
+      update_league_all(params['region'], params['id'])
```

Comment:
Are you going to `return` after this line? Otherwise it's going to run the `render json` code twince